lib/jackalope/tortoise_client.ex

```diff
@@ -151,6 +154,12 @@ defmodule Jackalope.TortoiseClient do
     {:noreply, state}
   end
 
+  def handle_continue(:start_telemetry, state) do
+    Logger.warn("STARTING TELEMETRY")
```

Comment:
Do we want to log this?

Reply:
Sorry. Still testing. I put the PR in draft mode.